flask/test/model_tests/test_box.py

```diff
@@ -30,5 +30,5 @@ def test_box_model(
     assert queried_box_dict["items"] == default_box["items"]
     assert queried_box_dict["location"]["id"] == default_box["location"]
     assert queried_box_dict["product"]["id"] == default_product["id"]
-    assert queried_box_dict["product"]["product_gender"] == default_product_gender
-    assert queried_box_dict["product"]["product_category"] == default_product_category
+    assert queried_box_dict["product"]["gender"] == default_product_gender
```

Comment:
here we do want to keep the differentiation between `product_gender `and `gender`. Gender will be used to refer to beneficiaries down the line, whereas product_gender just refers to products.

Reply:
This means a product can have both a gender and a product-gender?

Reply:
Ok I see. For the model fields I try to follow the convention of having little duplication in the names. Given a `Product` instance, the `gender` field is used as `product.gender` hence IMHO from the context it can be distinguished from e.g. `beneficiary.gender`, and the duplication `product.product_gender` is not necessary.